.nvmrc

```diff
@@ -1 +1 @@
-16.14.0
\ No newline at end of file
+18.12.0
```

Comment:
I won't lie - updating the node version our app uses scares me deeply 🤣

Reply:
Yeah this change literally breaks typeorm and needs reverted. `yarn set version berry` seems to depend on this version now as it flags the following error (on the develop branch too)

Usage Error: This tool requires a Node version compatible with >=18.12.0 (got 16.14.0). Upgrade Node, or set `YARN_IGNORE_NODE=1` in your environment.

Reply:
As discussed, lets revert the node version until we have e2e tests in place to have some sort of confidence. Will do some digging on the pipeline flow, but can go with that fix for now to get around it